### ISSUE TITLE
[UWP][Image] [Image doesn't include spacing]

### ISSUE THREAD

**Issue by adamhewitt627** [microsoft/AdaptiveCards#4035]
# Platform
* UWP

# Version of SDK
```xml
<PackageReference Include="AdaptiveCards" Version="1.2.4" />
<PackageReference Include="AdaptiveCards.Rendering.Uwp" Version="1.2.8" />
```

# Details

The `"Image"` element doesn't include any spacing set on it, but you will see spacing on a parent `ColumnSet`.

## Reproduction
```json
{
    "type": "AdaptiveCard",
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.2",
    "body": [
        {
            "text": "Here's a heading",
            "type": "TextBlock",
            "weight": "Bolder"
        },
        {
            "type": "ColumnSet",
            "spacing": "ExtraLarge",
            "columns": [
                {
                    "type": "Column",
                    "width": "stretch",
                    "items": [
                        {
                            "type": "Image",
                            "url": "https://img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE1Mu3b?ver=5c31"
                        }
                    ]
                }
            ]
        },
        {
            "type": "Image",
            "url": "https://img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE1Mu3b?ver=5c31",
            "spacing": "ExtraLarge"
        },
        {
            "text": "And some text below",
            "type": "TextBlock",
            "wrap": true
        }
    ]
}
```

![image](https://user-images.githubusercontent.com/3036550/82347611-b8148a80-99ac-11ea-85e6-91e6afed20c8.png)

**Comment by shalinijoshi19** [comment]
Hmm @paulcam206 quick stab here? I ensured it does not repro on the JS renderer so indeed looks like spacing isnt being honored on the UWP stack (both Image and ImageSet do inherit the Spacing property per the schema)

**Comment by paulcam206** [comment]
sure, I'll take a look :)

**Comment by paulcam206** [comment]
for what it's worth, this doesn't repro on `master`, still digging :)

**Comment by paulcam206** [comment]
has to do with `SetSeparatorVisibility`

**Comment by shalinijoshi19** [comment]
Thanks @paulcam206! If it doesn't repro on master, should we just wait until our July 2.0 release then for this to be addressed vs porting over as a 1.2 patch?
